Add external links to the Event page

torsava opened the pull request:
Fixes issue #38

torsava commented:
I chose the formulations:

    External link for this event
    External links for this event:  [ 1 ]  [ 2 ] ...

However, if you can think of a better ones, I'm open to suggestions.

torsava commented:
**Changes:**
- Changed the position to the very bottom of the page.
- Updated the English text to a better sounding one.
- Added a Czech translation.
- Entries are now shown using their base URL, not a number.

encukou commented on pyvocz/templates/event.html:
```diff
@@ -125,6 +125,20 @@
+            {{ tr('Jinde na webu:', 'Elsewhere on the web:') }}
+            {% for link in event.links|sort(attribute='index') %}
+                <a href="{{ link.url }}">{{ urlparse(link.url).netloc }}</a>
+                {%- if loop.index < event.links|length -%}
```
You can use `loop.last`, see http://jinja.pocoo.org/docs/2.9/templates/#for

torsava replied:
Done!

torsava commented:
**Changes:**
- Used loop.last for simplicity
- Moved the links below the description for better visibility and not to be lumped together with the location
- Removed 'www.' from the links if present.

encukou commented:
Looks great, thanks!
